### PR TITLE
Add missing options to the settings page

### PR DESCRIPTION
**Pull request by osbre**
As per https://docs.simpleanalytics.com/create-plugin#system-plugins

Closes https://github.com/simpleanalytics/wordpress-plugin/issues/3
Closes https://github.com/simpleanalytics/wordpress-plugin/issues/5

### CODE REVIEW

File: simple-analytics.php [hunk 1]
```diff
@@ -17,25 +17,39 @@
...
-	exit;
-}
+namespace SimpleAnalytics;
 
-function simpleanalytics_noscript() {
-	$domain = get_option( 'simpleanalytics_custom_domain' ) ?: 'queue.simpleanalyticscdn.com';
+defined('ABSPATH') || exit;
 
-	echo '<noscript><img src="https://' . $domain . '/noscript.gif" alt="" referrerpolicy="no-referrer-when-downgrade"></noscript>' . "\n";
+function should_collect_analytics(): bool
```
Comment (adriaandotcom) [inline simple-analytics.php:24]: Nice work already :) I know it's a draft but..

Don't forget that the is_logged_in is not the only thing that effects should_collect_analytics. People should be able to turn that behavior off via the UI. Default is on.

Comment (adriaandotcom) [reply simple-analytics.php:24]: Something I found online:

2024-03-05-15h43-screenshot@2x

The IPs are a nice to have.

Comment (osbre) [reply simple-analytics.php:24]: Thanks! Will include it in the "advanced" tab

**Comment by adriaandotcom** [comment]
How is this process going @osbre?

**Comment by osbre** [comment]
@adriaandotcom Sorry, been busy. I think we'll have it by Monday. On the tech side, I found a better way to manage fields so the current will soon get updated/replaced.

**Comment by adriaandotcom** [comment]
Thanks @osbre! Can you share some screenshots on how it looks?

**Comment by osbre** [comment]
image


image

image

It will be ready for testing in ~2 days.

**Comment by adriaandotcom** [comment]
How is the plugin going @osbre?

**Pull request merged**